src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -388,4 +390,10 @@ public void repoConfig_overrideKeyword_success() throws ParseException, IOExcept
         Assert.assertTrue(config.isIgnoreGlobListOverriding());
         Assert.assertTrue(config.isIgnoreCommitListOverriding());
     }
+
+    @Test (expected = IOException.class)
+    public void repoConfig_zeroValidRecords_fail() throws IOException {
```

Comment:
Let's follow the existing naming convention: `repoConfig_zeroValidRecords_throwsIOException()`

Reply:
Thanks, I will update the name.
However, in our project Checkstyle configuration (in `config/checkstyle/checkstyle.xml`), we do not allow variable names to have more than 2 consecutive capital letters.

I will use the name `repoConfig_zeroValidRecords_throwsIoException`